.gitignore

```diff
@@ -0,0 +1,3 @@
+# .gitignore
+node_modules/
+test.md
```

Comment:
I am afraid your test.md file is not ignored properly. 👎  Consider reviewing this 😺